### PR TITLE
mv vrf foundry tests

### PR DESCRIPTION
**Pull request by RensR**
Move VRF Foundry tests to the correct location

### CODE REVIEW

File: contracts/foundry.toml [hunk 1]
```diff
@@ -25,7 +25,7 @@ gas_price = 3_000_000_000 # 3 gwei
 [profile.vrf]
 optimizer_runs = 1_000
 src = 'src/v0.8/vrf'
-test = 'test/v0.8/foundry/vrf' # skips tests for no VRF foundry tests
+test = 'test/v0.8/vrf/test'
```
Comment (kidambisrinivas) [inline contracts/foundry.toml:28]: Should this be 'src/v0.8/vrf/test'?

File: contracts/src/v0.8/vrf/test/VRFV2Plus.t.sol [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-import {MockLinkToken} from "../../../../src/v0.8/mocks/MockLinkToken.sol";
-import {MockV3Aggregator} from "../../../../src/v0.8/tests/MockV3Aggregator.sol";
-import {ExposedVRFCoordinatorV2_5} from "../../../../src/v0.8/vrf/dev/testhelpers/ExposedVRFCoordinatorV2_5.sol";
-import {VRFCoordinatorV2_5} from "../../../../src/v0.8/vrf/dev/VRFCoordinatorV2_5.sol";
-import {SubscriptionAPI} from "../../../../src/v0.8/vrf/dev/SubscriptionAPI.sol";
-import {BlockhashStore} from "../../../../src/v0.8/vrf/dev/BlockhashStore.sol";
-import {VRFV2PlusConsumerExample} from "../../../../src/v0.8/vrf/dev/testhelpers/VRFV2PlusConsumerExample.sol";
-import {VRFV2PlusClient} from "../../../../src/v0.8/vrf/dev/libraries/VRFV2PlusClient.sol";
-import {VRFTypes} from "../../../../src/v0.8/vrf/VRFTypes.sol";
+import "./BaseTest.t.sol";
```
Comment (kidambisrinivas) [inline contracts/src/v0.8/vrf/test/VRFV2Plus.t.sol:3]: Should we update any CI checks for this move?

Comment (RensR) [reply contracts/src/v0.8/vrf/test/VRFV2Plus.t.sol:3]: CI uses foundry.toml to run tests, that's why the test location has been updated in that file

File: contracts/foundry.toml (hunk 1 above)
Comment (RensR) [reply contracts/foundry.toml:28]: Yes that is the new location

**Review by kidambisrinivas — APPROVED** [review]

**Pull request merged**